src/m365/tenant/commands/report/report-office365activationcounts.ts

```diff
@@ -0,0 +1,78 @@
+import { Logger } from '../../../../cli';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: GlobalOptions;
+}
+
+class TenantReportOffice365activationcountsCommand extends GraphCommand {
+  public get name(): string {
+    return commands.REPORT_OFFICE365ACTIVATIONCOUNTS;
+  }
+
+  public get description(): string {
+    return 'Get the count of Microsoft 365 activations on desktops and devices.';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    const endpoint: string = `${this.resource}/v1.0/reports/getOffice365ActivationCounts`;
+    this.executeReport(endpoint, logger, args.options.output, cb);
+  }
+
+  protected executeReport(endPoint: string, logger: Logger, output: string | undefined, cb: () => void): void {
+    const requestOptions: any = {
+      url: endPoint,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    request
+      .get(requestOptions)
+      .then((res: any): void => {
+        let content: string = '';
+        const cleanResponse = this.removeEmptyLines(res);
+
+        if (output && output.toLowerCase() === 'json') {
+          const reportData: any = this.getReport(cleanResponse);
+          content = JSON.stringify(reportData);
```

Comment:
In JSON output we need to pass the original object to CLI. If we serialize it first, we'll get an escaped JSON string rather than a correct JSON string than can be parsed with jq or PowerShell